packages/check-time/extra/check-time.c

```diff
@@ -0,0 +1,41 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <sys/timex.h>
+#include <string.h>
+#include <errno.h>
+
+#define USEC_PER_MSEC 1000L
+#define MAX_EST_ERROR_US 100L * USEC_PER_MSEC /* 100 millisecond */
+
+int main() {
+	struct timex tx;
+	int rc;
+	long long int error;
+
+	if (getenv("IGNORE_CHECK_TIME")) {
```

Comment:
Eh, I think that this is overkill. If IGNORE_CHECK_TIME is set to anything (1, foo, pants) we should ignore it.